br/pkg/lightning/restore/restore.go

```diff
@@ -2299,6 +2299,8 @@ type deliveredKVs struct {
 	columns []string
 	offset  int64
 	rowID   int64
+
+	realOffset int64
```

Comment:
Maybe add some comment, indicating that `realOffset` is only used in compressed file scenarios.

Reply:
addressed in https://github.com/pingcap/tidb/pull/39219/commits/32071eda886a93da125f95529c2a4cf334864ade